tests/users/test_views.py

```diff
@@ -0,0 +1,136 @@
+import pytest
+from django.core import mail
+from django.urls import reverse
+from django.test import Client
+
+from vitrina.users.models import User
+
+
+@pytest.fixture
+def user():
+    user = User.objects.create_user(email="test@test.com", password="test123")
+    return user
+
+
+@pytest.mark.django_db
+def test_login_with_wrong_credentials(client: Client, user):
```

Comment:
```suggestion
def test_login_with_wrong_credentials(client: Client, user: User):
```

User type annotations.